document/FileFormat.mdk

```diff
@@ -2602,6 +2602,8 @@ This property should be absent only if the tool does not have physical location
 
 NOTE: This could happen, for example, if this `annotatedCodeLocation` occurs within a binary for which
 the tool does not have associated symbol information.
+It might also happen if the value of the `kind` property (&sect;[#annotatedCodeLocation-kind]) is `"functionExit"`,
```

Comment:
>also happen [](start = 9, length = 11)

 A kind of 'continuation', may also have no location (it may only track global variable state changes).